subsys/dfu/src/dfu_target_modem.c

```diff
@@ -0,0 +1,224 @@
+#include <zephyr.h>
+#include <flash.h>
+#include <dfu/dfu_target.h>
+#include <net/socket.h>
+#include <nrf_socket.h>
+#include <logging/log.h>
+#include "dfu_target_modem.h"
+
+LOG_MODULE_REGISTER(dfu_target_modem, CONFIG_DFU_TARGET_LOG_LEVEL);
+
+#define DIRTY_IMAGE 2621440
```

Comment:
Can this be hex instead? `0x280000`? It would look better IMO.

Reply:
Sure.